src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -113,18 +161,40 @@ class NotifyService(
     }
 
     if (emailAddress.isBlank()) {
-      log.info("Blank email address: Did not send notification for $templateId ref $reference")
+      log.info("Notification - blank email address: Did not send for template ID $templateId ref $reference")
       return
     }
 
     try {
       client.sendEmail(templateId, emailAddress, values, reference)
     } catch (e: NotificationClientException) {
-      log.error("Email notification failed - templateId $templateId to $emailAddress", e)
+      log.error("Notification failed - templateId $templateId to $emailAddress", e)
     }
   }
 
+  private fun getContacts(creatorEmail: String, creatorName: String, comEmail: String, comName: String): List<Contact> {
+    val contacts = mutableListOf<Contact>()
+    val creatorIsBlank = creatorEmail.isEmpty()
+    val comIsBlank = comEmail.isEmpty()
+    if (creatorIsBlank && !comIsBlank) {
+      contacts.add(Contact(comEmail, comName))
+    } else if (comIsBlank && !creatorIsBlank) {
+      contacts.add(Contact(creatorEmail, creatorName))
+    } else if (creatorEmail == comEmail) {
+      contacts.add(Contact(creatorEmail, creatorName))
```

Comment:
You could combine this into one statement:
else if((comIsBlank && !creatorIsBlank) || creatorEmail == comEmail)

It saves a duplicate line of code, but adds more complexity to the if statement. So quite subjective.

Reply:
Yes, I thought of doing it like that, but settled on the above way and being 'easier to read'.